### PR TITLE
[ViewingRoom] Make responsive

### PR DESCRIPTION
**Pull request by damassi**
This PR adds responsivity to the new Viewing Rooom app: 

viewing-room-responsive

### CODE REVIEW

**Comment by damassi** [comment]
@dzucconi Gonna merge this and keep going, but can loop back to address any comments.

File: src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx [hunk 1]
```diff
@@ -1,12 +1,79 @@
...
   return (
-    <HeaderContainer>
-      <ImageArea>
+    <>
+      <Media greaterThanOrEqual="sm">
+        <ViewingRoomHeaderLarge />
+      </Media>
+      <Media lessThan="sm">
+        <ViewingRoomHeaderSmall />
+      </Media>
```
Comment (damassi) [inline src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx:14]: These layouts are different enough to warrant two different components (I think -- might revise this later)

**Comment by dzucconi** [comment]
👍 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`v26.34.3`](https://github.com/artsy/reaction/releases/tag/v26.34.3) :rocket:

**Pull request merged**